api/src/main/java/org/xnio/XnioWorker.java

```diff
@@ -110,7 +110,7 @@ protected XnioWorker(final Xnio xnio, final ThreadGroup threadGroup, final Optio
             workerName = "XNIO-" + seq.getAndIncrement();
         }
         name = workerName;
-        taskQueue = new LinkedBlockingQueue<Runnable>();
+        taskQueue = new DisruptorBlockingQueue<Runnable>(50000);
```

Comment:
What \is the rationale behind 50000? Can it be more or less and what will be the impact?